pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3694,6 +3721,9 @@ int BPF_KPROBE(trace_ret_do_mmap)
     save_to_submit_buf(&p.event->args_buf, &prot, sizeof(unsigned long), 8);
     save_to_submit_buf(&p.event->args_buf, &mmap_flags, sizeof(unsigned long), 9);
 
+    if (!evaluate_data_filters(&p, 1))
```

Comment:
This should be checked only `if (file != NULL)`.

Reply:
Good point... think its better to only call evaluate_data_filters after `if (file != NULL)`.

Soon, when we implement more than one index filter, won't be necessary to pass index as parameter to `evaluate_data_filters` in this way will be possible to remove `if (file != NULL)`.